app/controllers/application_controller.rb

```diff
@@ -21,6 +22,14 @@ def not_authorized!
 
   private
 
+  def set_session_expiry
+    session_length = Rails.application.config.session_options[:expire_after]
+    cookies['session_expiry'] = {
+        value: (Time.now + session_length).to_f
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.